test/new-e2e/README.md

```diff
@@ -21,3 +21,18 @@ The `agent-sandbox: test current file` can be used to launch test on a file with
 You need to open the `new-e2e` folder
 
 > **Note** > `go.work` file is currently ignored in `datadog-agent`
+
+## Running new-e2e tests locally
+
+### Installer tests
+
+Installer tests are tests owned by Agent Platform, that mostly test the installation of the agent on different platforms with different methods. These tests are defined in `./tests/agent-platform`.
+The following are considered as installer tests:
+- `./tests/agent-platform/install-script`: Test the installation of the agent using the install script
+- `./tests/agent-platform/step-by-step`: Test the installation of the agent without the install script, following the manual installation described on the official documentation ([for example](https://app.datadoghq.com/account/settings/agent/latest?platform=debian))
+- `./tests/agent-platform/upgrade`: Test the upgrade of the agent using the install script
+
+All these tests can be executed locally with the following command:
+`aws-vault exec sso-agent-sandbox-account-admin -- go test <test folder path> -v -args -osversion '<comma separated os version list>' -platform '<debian/centos/suse/ubuntu>' -arch <x86_64/arm64>`
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Wonder if we should use the inv task instead, as described in internal docs. Not sure if it allows passing extra params to `gotestsum`